database/__init__.py

```diff
@@ -206,6 +229,23 @@ def edit(token):
     pass
 
 
+@app.route('/view_data/<token>')
+def view_data(token):
+    try:
+        email = ts.loads(token, salt='request_data-key')
+    except SignatureExpired:
+        abort(404)
```

Comment:
`abort()` is not defined, is this correct?

Reply:
Mh that should be coming from flask (http://flask.pocoo.org/docs/1.0/api/#flask.abort)

Reply:
`from flask import abort`

Reply:
Bzw. einfach abort oben bei den `from flask import ...` hinzufügen